README.md

```diff
@@ -34,17 +34,18 @@ The first three steps are needed for both Android and iOS devices.
 1. Create an environment variable file named `.env` in the root directory
     ```
     # Parse server configuration
-    # Dogfood Server URL http://130.211.12.3/parse
+    # Dogfood Server URL https://wallet.app/v1

```

Comment:
If RWALLET_ENV is not set or Production, the actual url will remain as it is, otherwise it, otherwise the RWALLET_ENV will be lowercase and used as a subdomain.
->
If RWALLET_ENV value is "Production", or not set, the final server URL value is the same as Server URL such as "https://wallet.app/v1"; otherwise, the RWALLET_ENV value will be prepended to Server URL in lower case as a sub-domain. For example, if RWALLET_ENV is "Dogfood" and Server URL is "https://wallet.app/v1", the final URL is https://dogfood.wallet.app/v1.